src/commands/add.rs

```diff
@@ -94,6 +90,10 @@ pub async fn command(args: Args, _json: bool) -> Result<()> {
             post_graphql::<mutations::PluginCreate, _>(&client, configs.get_backboard(), vars)
                 .await?;
             spinner.finish_with_message(format!("Created {plugin}"));
+        } else {
+            println!("Creating {}...", plugin);
+            post_graphql::<mutations::PluginCreate, _>(&client, configs.get_backboard(), vars)
```

Comment:
Doesn't have to be addressed in this PR, but we should really find a way to combine the tty/non-tty flows. Having to call the mutation twice per command is not great at all.